src/utils/SmartParser.tsx

```diff
@@ -0,0 +1,461 @@
+/**
+ * @name SmartParser
```

Comment:
Je pense qu'il vaut mieux le nommer `smartParse.ts`; normalement les majuscules sont réservées aux composants (fichiers .tsx)

Reply:
C'est noté, j'ai modifié ✅